Modules/Sources/PointOfSale/Models/PointOfSaleAggregateModel.swift

```diff
@@ -2,6 +2,7 @@ import CocoaLumberjackSwift
 import Foundation
 import Combine
 import Observation
+import enum Hardware.CardReaderSoftwareUpdateState
```

Comment:
Thoughts on importing `Hardware` here? Should we keep it in Yosemite and inject it through somewhere else into the aggregate model? 🤔

Reply:
You can import it from Yosemite as there's a typealias already in Model.swift
We shouldn't import Hardware above Yosemite
```suggestion
import enum Yosemite.CardReaderSoftwareUpdateState
```